src/status_im/subs.cljs

```diff
@@ -1228,6 +1254,21 @@
  (fn [currency-id]
    (get constants/currencies currency-id)))
 
+(defn filter-recipient-favs
```

Comment:
same as `filter-recipient-accounts`?

Reply:
just a coincidence, one may change in the future